src/main/java/org/jabref/gui/push/PushToApplicationSettings.java

```diff
@@ -19,7 +19,8 @@
     protected AbstractPushToApplication application;
     private DialogService dialogService;
 
-    public GridPane getJFXSettingPane(int n) {
+    public GridPane getJFXSettingPane(int n, DialogService ds) {
```

Comment:
Can you please try to pass the dialog service as a constructor argument (reason: dependencies should be specified in the constructor). It might be the case, that it is not possible to pass it as a constructor argument (without major refactoring). In this case, we can merge it as it is.

Reply:
Should be simple here as the Manager class for PushToApplication already has the dialogService

Reply:
Thought so too, but "error: non-static variable dialogService cannot be referenced from a static context"
The Managerclass seems to be static... lol

Reply:
Ah yeah, overlooked it. I think it makes sense to refactor the static mehtod to an instance method.
And initilaize the PushToApplicationManager in the ExternalTabs in the constructor.